src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    public Square(Colors color, double side) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Consider creating separate variables for color and side and then pass them to the constructor for better code readability.